spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -1874,6 +1917,48 @@ public void returned(AmqpMessageReturnedException e) {
 	}
 
 	/**
+	 * A wrapper for the contents of a message delivery.
+	 */
+	public static class Delivery {
```

Comment:
This looks similar to the `BlockingQueueConsumer.Delivery`.
Can't we extract it as a top-level class now ?